manticore/platforms/evm.py

```diff
@@ -2103,6 +2154,13 @@ def _process_pending_transaction(self):
         if failed:
             self._close_transaction('TXERROR', rollback=True)
 
+        if caller not in self._world_state:
+            self._world_state[caller] = {}
+        if 'nonce' not in self._world_state[caller] or self._world_state[caller]['nonce'] is None:
+            self._world_state[caller]['nonce'] = 1
+        else:
+            self._world_state[caller]['nonce'] += 1
```

Comment:
Since we already access `[caller]['nonce']` then maybe:
```python
self._world_state[caller]['nonce'] = self._world_state[caller].get('nonce', 0) + 1

Reply:
Is `caller not in self._world_state` the only situation when `'nonce'` key is not there?

If so we can do:
```python
        if caller not in self._world_state:
            self._world_state[caller] = {'nonce': 1}
        else:
            self._world_state[caller]['nonce'] += 1
```